arm/Microsoft.Automation/automationAccounts/.parameters/encr.parameters.json

```diff
@@ -0,0 +1,207 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "name": {
+            "value": "<<namePrefix>>-wd-aut-encr-001"
+        },
+        // "schedules": {
+        //     "value": [
+        //         {
+        //             "name": "TestScheduleEncr",
+        //             "startTime": "",
+        //             "expiryTime": "9999-12-31T23:59:00+00:00",
+        //             "interval": 15,
+        //             "frequency": "Minute",
+        //             "timeZone": "Europe/Berlin",
+        //             "advancedSchedule": {}
+        //         }
+        //     ]
+        // },
+        "encryptionKeySource" : {
+            "value" : "Microsoft.Keyvault"
+        },
+        "encryptionUserAssignedIdentity": {
+            "value": "/subscriptions/<<subscriptionId>>/resourcegroups/validation-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/adp-<<namePrefix>>-az-msi-x-001" // this identity needs to be one of the identities defined in userAssignedIdentities section
+        },
+        "keyName" : {
+                "value" : "keyEncryptionKey"
+        },
+        "keyvaultUri" : {
+                    "value" : "https://adp-carml-az-kv-nopr-002.vault.azure.net/"
+        },
+        "keyVersion" : {
+                    "value" : "9917c14be51d4d93b37218de7d326f60"
+        },
+        // "modules": {
```

Comment:
If we don't test the modules, I'd suggest to remove them. same for the other parameters

Reply:
Removed and tested (They were commented for testing purposes)